scripts/solc-bump.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const readline = require('readline');
+
+const PACKAGES_DIR = `${__dirname}/../packages`;
+
+/**
+ * Load packages depending on `solc`
+ **/
+function loadSolcDependentPackages() {
+  const packages = fs.readdirSync(PACKAGES_DIR);
+  return packages.filter(packageName => {
+    // parse the package.json file for `packageName`
+    const packagePath = `${PACKAGES_DIR}/${packageName}/package.json`;
+    const packageJson = JSON.parse(fs.readFileSync(packagePath, 'utf8'));
+
+    // operate if the solc dependency exists
+    return packageJson
+      && packageJson.dependencies
+      && !!packageJson.dependencies['solc'];
+  });
+}
+
+/**
+ * Use child_process to run `npm install` in `packageName` synchronously
+ **/
+function updatePackages(packageNames) {
+  packageNames.forEach(packageName => {
+    console.log(`Updating ${packageName}`);
+    execSync('npm install solc@latest', {
```

Comment:
Are requirements for this script to save the value to the `package.json`? It seems that you would need `--save` for that.

Reply:
npm install actually saves to dependencies by default, but I added explicity `--save` and `--save-dev` with `devDependencies` support